.gitlab-ci.yml

```diff
@@ -611,14 +614,19 @@ publish_community_operators:
       ci.datadog-operator.redhat-registry-token --with-decryption --query "Parameter.Value" --out text > ~/.redhat/auths.json
     - cd $CI_PROJECT_DIR
     - make bundle-redhat
-    # configure git and github-cli
-    - git config --global user.email $GITLAB_USER_EMAIL
-    - git config --global user.name $GITLAB_USER_NAME
```

Comment:
that depends on if you want to remove it. This should not have any effect on DD Octo STS, I believe. But not certain. Maybe you have to set it later to the DD Octo STS user 🤷

Reply:
added back `GITLAB_USER_EMAIL`